invenio_circulation/api.py

```diff
@@ -53,6 +51,16 @@ def get_record_by_pid(cls, pid, with_deleted=False):
         persistent_identifier, record = resolver.resolve(str(pid))
         return record
 
+    def attach_item_ref(self, data):
+        """Attach item reference."""
+        item_ref_builder = current_app.config.get(
```

Comment:
Just a cosmetic comment, could we just have a `self.item_ref_builder` so we would avoid reading the config every time?